src/cmake/thirdparty/SetupAxomThirdParty.cmake

```diff
@@ -91,10 +91,21 @@ endif()
 #------------------------------------------------------------------------------
 if (HDF5_DIR)
     include(cmake/thirdparty/SetupHDF5.cmake)
-    blt_register_library( NAME      hdf5
-                          INCLUDES  ${HDF5_INCLUDE_DIRS}
-                          LIBRARIES ${HDF5_LIBRARIES}
-                          TREAT_INCLUDES_AS_SYSTEM ON)
+    if(WIN32 AND TARGET hdf5::hdf5-shared)
```

Comment:
This is a bit confusing to me.

Do we currently support ``static`` and ``shared`` builds of ``hdf5`` on Windows, or only ``shared`` builds?

If it is the latter (which is my understanding). is the ``else()`` clause ( ``hdf::hdf-static``)  only there as a placeholder?


Could you please add a comment to clarify the current situation?

Reply:
Added a comment to clarify that the `else` clause is a placeholder